lib/kuniri/language/ruby/function_behavior_ruby.rb

```diff
@@ -64,15 +64,18 @@ def handling_default_parameter(pLine)
 
       # Override
       def remove_unnecessary_information(pLine)
-        return pLine.gsub(/\s+|\(|\)/, '') if pLine =~ /\s+|\(|\)/
-        return pLine
+        return pLine if pLine !~ /\s+|\(|\)/
+
+        nLine = pLine.gsub(/\s+|\(|\)/, '')
+
+        nLine != '' ? nLine : nil
       end
 
       # Override
       def handling_parameter(pLine)
         # Handling with parenthesis and without it.
-        if pLine =~ /\(.+\)/
-          partial = get_parameters(pLine, /\(.+\)/)
+        if pLine =~ /\(.*\)/
```

Comment:
For the case when there is space between method name and parenthesis:
``
def method ()
``
It would not match the regex:
``
/\(.+\)/
``
And would be handled by the case:
``
elsif pLine =~ /def\s+\w+[\s]+(.+)/
``
Which works, but doesn't seems right.